internal/printer/printer_test.go

```diff
@@ -181,13 +182,9 @@ import * as ns from '../components';
   </body>
 </html>`,
 			want: want{
-				frontmatter: []string{
-					`import * as ns from '../components';
-
-import * as $$module1 from '../components';`,
```

Comment:
Cleanup: you no longer need to specify these! 🎉 They’re just auto-generated from `metadata.modules` below.